qemu/tests/qemu_guest_agent.py

```diff
@@ -836,31 +841,32 @@ def gagent_check_fsfreeze(self, test, params, env):
         2) Freeze the FS.
         3) Check the FS is frozen from both guest agent side and guest os side.
         4) Thaw the FS.
+        5) Check the FS is unfrozen from both guest agent side and guest os side.
 
         :param test: kvm test object
         :param params: Dictionary with the test parameters
         :param env: Dictionary with test environmen.
         """
-        error.base_context("Check guest agent command 'guest-fsfreeze-freeze'",
-                           logging.info)
-        error_context.context("Verify FS is thawed and freeze the FS.")
-
+        error_context.context("Check guest agent command "
+                              "'guest-fsfreeze-freeze/thaw'",
+                              logging.info)
         try:
             expect_status = self.gagent.FSFREEZE_STATUS_THAWED
             self.gagent.verify_fsfreeze_status(expect_status)
         except guest_agent.VAgentFreezeStatusError:
             # Thaw guest FS if the fs status is incorrect.
             self.gagent.fsthaw(check_status=False)
 
-        self._action_before_fsfreeze(test, params, env)
+        self._action_before_fsfreeze()
+        error_context.context("Freeze the FS.")
```

Comment:
lack of 'logging.info'.

Reply:
it's ok without logging.info like line867. 
05:12:12 INFO | Context: Freeze the FS. --> Verfiy FS is frozen in guest.